src/guides/v2.4/test/svc/semver_test_execution.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This file is identical (it will be if the reference to the development branch is removed) to the 2.3 version - therefore it should be created as a symlink, not a separate file.